### PR TITLE
Bucket sdk

### PR DESCRIPTION
**Pull request by bernardomp**
Generated python and nodejs sdk for Bucket component

### CODE REVIEW

File: provider/cmd/pulumi-resource-cloud-toolkit-aws/storage/config.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import * as pulumi from "@pulumi/pulumi";
+import defaultsDeep from "lodash.defaultsdeep";
+
+export interface BucketArgs {
```
[the remark this answers is not in the document]
Comment (bernardomp) [reply provider/cmd/pulumi-resource-cloud-toolkit-aws/storage/config.ts]: Fixed

File: provider/cmd/pulumi-resource-cloud-toolkit-aws/storage/config.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import * as pulumi from "@pulumi/pulumi";
```
[the remark this answers is not in the document]
Comment (bernardomp) [reply provider/cmd/pulumi-resource-cloud-toolkit-aws/storage/config.ts]: Fixed

File: provider/cmd/pulumi-resource-cloud-toolkit-aws/storage/bucket.ts [hunk 3]
```diff
@@ -0,0 +1,530 @@
...
+import { BucketArgs, validateConfig } from "./config";
+
+export { BucketArgs };
+
+/**
+ * Bucket component
+ *
+ * @extends {pulumi.ComponentResource}
+ */
+export class Bucket extends pulumi.ComponentResource {
```
[the remark this answers is not in the document]
Comment (bernardomp) [reply provider/cmd/pulumi-resource-cloud-toolkit-aws/storage/bucket.ts:12]: Fixed

File: sdk/nodejs/index.ts [hunk 4]
```diff
@@ -17,17 +17,11 @@ utilities.lazyLoad(exports, ["Example"], () => require("./example"));
 utilities.lazyLoad(exports, ["Provider"], () => require("./provider"));
 
 // Export sub-modules:
-import * as email from "./email";
-import * as kubernetes from "./kubernetes";
-import * as landingzone from "./landingzone";
-import * as serverless from "./serverless";
```
Comment (RedwineOS) [inline sdk/nodejs/index.ts:23]: I think this might be a mistake when rebasing.

**Review by RedwineOS — COMMENTED** [review]
Seems that the CI failed due to mismatched dependencies. I have a comment over a small code change. The rest looks quite good and ready to merge.

**Comment by RedwineOS** [comment]
Also, the `schema.yaml` wasn't updated.

**Comment by bernardomp** [comment]
image
Pending to review errors

**Comment by bernardomp** [comment]
Not needed

**Pull request closed without merging**